main/sceneBuilder.cpp

```diff
@@ -0,0 +1,75 @@
+#include "gtest/gtest.h" // https://google.github.io/styleguide/cppguide.html#Namespaces
+#include "main/sceneBuilder.h"
+#include "obj/generators.h"
+#include "obj/objects.h"
+
+using core::Ray;
+using core::RayHitData;
+using core::Vec3;
+using generators::PointSource;
+using objects::Object;
+using objects::TriangleObj;
+
+const double kSkipFreq = 1000;
+generators::FakeRandomGen fakeRand;
```

Comment:
Do not put it here, put it in a fixture and use TEST_F. This way generator will be reset on every test automatically.

Reply:
I need to read more about google test framework then :)